external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -46,6 +46,20 @@ trait V2JDBCTest extends SharedSparkSession {
     assert(msg.contains("Cannot update missing field bad_column"))
   }
 
+  def testRenameColumn(tbl: String): Unit = {
+    sql(s"CREATE TABLE $tbl (ID STRING NOT NULL, ID1 STRING NOT NULL) USING _")
+    sql(s"ALTER TABLE $tbl RENAME COLUMN ID TO ID2")
+    val t = spark.table(s"$tbl")
+    val expectedSchema = new StructType().add("ID2", StringType, nullable = true)
+      .add("ID1", StringType, nullable = true)
+    assert(t.schema === expectedSchema)
+    // Rename to already existing column
```

Comment:
This can be put in the `test("SPARK-33034: ALTER TABLE ... rename column")`, as the behavior should always be the same.

Reply:
done